src/components/Button.js

```diff
@@ -1,23 +1,21 @@
-import React, { Component } from 'react';
+import React from 'react';
 import PropTypes from 'prop-types';
 
-export class Button extends Component { // eslint-disable-line
-  render() {
-    return (
-      <button onClick={this.props.handleClick} className={this.props.class} type="button">{this.props.text}</button> // eslint-disable-line
-    );
-  }
-}
+export const Button = ({ handleClick, addClass, text }) => { // eslint-disable-line
+  return (
+    <button onClick={handleClick} className={addClass} type="button">{text}</button>
+  );
+};
```

Comment:
- It would be great to remove the eslint-disable-line comment to get more clean your code.